net/udp-broadcast-relay-redux-openwrt/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2006-2014 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=udp-broadcast-relay-redux
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=$(PKG_SOURCE_VERSION)
+PKG_LICENSE:=GPL-2.0
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/udp-redux/udp-broadcast-relay-redux
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=671372938b55a186625a80516f86e8b9948c977a
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.xz
+PKG_MIRROR_HASH:=dc94a91a9ccf5ba07f7a2b7a9d766927fa074f55aa1449f460f28215dd2f5f2f
```

Comment:
I got this value by running the build and then running sha256sum against the tarball in the 'dl' dir.  Hopefully that's right?

Reply:
It's okay.

Another way is setting `PKG_MIRROR_HASH` to `skip`, then run `make package/<your package name>/check FIXUP=1`

Reply:
:+1:

Reply:
I see CI reports the correct HASH is `11cf8728f2b8e966f4f57032d817a889f680ed8e61afff35b52ca9c6789a03c6`.
Could you please have a check?